lib/utils/manifest-key-prefix-helper.js

```diff
@@ -20,6 +20,10 @@ const WebpackConfig = require('../WebpackConfig'); //eslint-disable-line no-unus
 module.exports = function(webpackConfig) {
     let manifestPrefix = webpackConfig.manifestKeyPrefix;
     if (null === manifestPrefix) {
+        if (null === webpackConfig.publicPath) {
+            throw new Error('publicPath is not set on WebpackConfig');
```

Comment:
This change looks unrelated

Reply:
It totally is :). When I didn't fully set up a new test case, I hit a bad "calling replace on null" error (the next line). So, I covered that edge case with a proper error. It won't affect real users - if they somehow were hitting this before, I've just made their error more clear :)